scripts/installer/auto-install.js

```diff
@@ -0,0 +1,90 @@
+// Run with:
+// ./msys2-x86_64-$(date +'%Y%m%d').exe --platform minimal --script auto-install.js -v
+// To specify the installation directory, add InstallPrefix="C:\custom_install\path\"
+// Currently it gets stuck on the last page.
+// To see graphically what's happening, remove "--platform minimal"
+
+var install_dir = installer.value("InstallPrefix")
+
+function Controller()
+{
+    print("(print) Hello Installer World!\n");
+    console.log("(console.log) Hello Installer World!\n");
```

Comment:
Can these strings pop up somewhere visible?

Reply:
I didn't check. I should probably remove them.